main/com.spotify/index.js

```diff
@@ -60,6 +61,10 @@ module.exports = class SpotifyDevice extends Tp.BaseDevice {
         this._state = new Map();
         this._queryResults = new Map();
         this._deviceState = new Map();
+
+        if (this.platform.type == "server") {
+            this.spotifyd = new spotifyd({ cacheDir: this.platform._cacheDir, username: this.state.id, device_name: this.state.id, token: this.accessToken });
```

Comment:
On occasion, we need to refresh access token. Usually that happens when we issue an API request, the request fail with a 401, so the internal OAuth machinery will call `refreshCredentials` on the device instance (an automatically injected method) which will do the token refresh, and eventually call `updateState` with the new access token and refresh token. Is that supportable with spotifyd?